src/utils/components/SSHAccess/utils.ts

```diff
@@ -68,9 +71,21 @@ export const createSSHService = async (vm: V1VirtualMachine, vmi: V1VirtualMachi
 
   const serviceResource = buildSSHServiceFromVM(vm, labelSelector);
 
-  await k8sCreate({
+  return await k8sCreate({
     model: ServiceModel,
     data: serviceResource,
     ns: namespace,
   });
 };
+
+export const getConsoleVirtctlCommand = (vmName: string, vmNamespace?: string) => {
+  let command = 'virtctl ';
+
+  if (vmNamespace) {
+    command += `-n ${vmNamespace} `;
+  }
+
+  command += vmName;
```

Comment:
Are you sure about the command `virtctl -n <vmNamespace> <vmName>` ? According to the doc, we are missing ` ssh ` there...

Reply:
Yes! I've installed an old version and there was no ssh command. Now I installed the 0.53 and there is!